test/unit/model/event/DeliveryCreatedEventTest.php

```diff
@@ -46,7 +47,7 @@ protected function setUp(): void
 
     public function testSerializeForWebhook()
     {
-        $event = new DeliveryCreatedEvent($this->deliveryMock);
+        $event = new DeliveryCreatedEvent($this->deliveryMock, $this->testResource);
```

Comment:
could u update the class with return type hints please?

Reply:
Done